src/Paramore.Brighter/NullOutboxArchiveProvider.cs

```diff
@@ -0,0 +1,31 @@
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Paramore.Brighter.Logging;
+
+namespace Paramore.Brighter
+{
+    /// <summary>
+    /// Use this archiver will result in messages just being deleted from the outbox and not stored
+    /// </summary>
+    public class NullOutboxArchiveProvider : IAmAnArchiveProvider
```

Comment:
LOL, send to null

Reply:
Well I had to ways to go, either write code in to give you an "Action" or default register you a don't write store. On the fence weather I like it, just know I don't currently hate it, thoughts @iancooper